libcxx/include/__utility/exception_guard.h

```diff
@@ -137,6 +137,11 @@ _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR __exception_guard<_Rollback> __make_exce
   return __exception_guard<_Rollback>(std::move(__rollback));
 }
 
+template <class _Rollback>
+_LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR __exception_guard_exceptions<_Rollback> __make_scoped_guard(_Rollback __rollback) {
```

Comment:
I guess `_LIBCPP_CONSTEXPR_SINCE_CXX20` should be used. Or perhaps we can use plain `constexpr` and guard this with `_LIBCPP_STD_VER >= 23`?

Reply:
Is there a reason why `_LIBCPP_CONSTEXPR_SINCE_CXX20` is needed in any of the class in this file?

Reply:
> Is there a reason why `_LIBCPP_CONSTEXPR_SINCE_CXX20` is needed in any of the class in this file?

Constexpr-ness of these classes relies on constexpr destructors, which is introduced in C++20 by [P0784R7](https://wg21.link/p0784r7).